### PR TITLE
feat: welcome nft banner

### PR DESCRIPTION
**Pull request by grabbou**
Fixes WEB-595

This is my first take on providing more device optimised assets by using `srcSet`.

### CODE REVIEW

File: src/nft/components/explore/WelcomeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import { X } from 'react-feather'
+import styled, { useTheme } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const Container = styled.div`
+  position: relative;
+  display: flex;
+  padding: 30% 24px 24px;
+  overflow: hidden;
+  height: fit-content;
```
Comment (grabbou) [inline src/nft/components/explore/WelcomeModal.tsx:14]: I am not particularly happy to set height as well as paddings in this case, but `fit-content` together with `overflow: hidden` was required to hide an overflow caused by an absolute background image. 

Sure, we could've just used a CSS background, but an equivalent of `src-set` for CSS backgrounds is still not widely supported.

File: src/nft/components/explore/WelcomeModal.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+`
+
+const Link = styled.a`
+  outline: none;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+`
```
Comment (grabbou) [inline src/nft/components/explore/WelcomeModal.tsx:32]: General question: is there a better way to handle links within the app? This looks like something you'd typically want to do for every link!

File: src/state/user/hooks.tsx [hunk 3]
```diff
@@ -117,6 +118,15 @@ export function useShowSurveyPopup(): [boolean | undefined, (showPopup: boolean)
...
 }
 
+export function useHideNFTWelcomeModal(): [boolean | undefined, () => void] {
+  const dispatch = useAppDispatch()
+  const hideNFTWelcomeModal = useAppSelector((state) => state.user.hideNFTWelcomeModal)
+  const hideModal = useCallback(() => {
+    dispatch(updateHideNFTWelcomeModal({ hideNFTWelcomeModal: true }))
+  }, [dispatch])
+  return [hideNFTWelcomeModal, hideModal]
+}
```
Comment (grabbou) [inline src/state/user/hooks.tsx:128]: Open to feedback if there's better way to do it! Based on other PRs, but it looks a bit verbose for just a single boolean!

Comment (lynnshaoyu) [reply src/state/user/hooks.tsx:128]: i think this is the standard way we do it, so lgtm

Comment (grabbou) [reply src/state/user/hooks.tsx:128]: Thanks! I was referring to your PR actually, but just couldn't bring the number up while writing this comment!

File: src/nft/components/explore/WelcomeModal.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/nft/components/explore/WelcomeModal.tsx:32]: Yes pls look at StyledInternalLink or StyledLink from src/theme/components/index.tsx

File: src/nft/pages/explore/index.tsx [hunk 4]
```diff
@@ -24,6 +25,7 @@ const ExploreContainer = styled.div`
 
 const NftExplore = () => {
   const setBagExpanded = useBag((state) => state.setBagExpanded)
+  const [isModalHidden, hideModal] = useHideNFTWelcomeModal()
```
Comment (lynnshaoyu) [inline src/nft/pages/explore/index.tsx:28]: smart🤓🧠

**Review by lynnshaoyu — APPROVED** [review]
pls just change the link to use one of the predefined Link Components but other than that LGTM

**Comment by grabbou** [comment]
@lynnshaoyu I changed it as per your suggestion, thanks for pointing this out for me! 

Important, there's an ongoing discussion on how this should look like on mobile which has to do with the way Modal component is rendered on mobile viewports. In case we decide to apply margins, I will send a follow up PR.

**Pull request merged**